benches/misc/coprocessor/codec/chunk/arrow.rs

```diff
@@ -23,14 +23,18 @@ impl Chunk {
             }
         }
 
-        match field_type.tp() {
+        match field_type.as_accessor().tp() {
             FieldTypeTp::Tiny
             | FieldTypeTp::Short
             | FieldTypeTp::Int24
             | FieldTypeTp::Long
             | FieldTypeTp::LongLong
             | FieldTypeTp::Year => {
-                if field_type.flag().contains(FieldTypeFlag::UNSIGNED) {
+                if field_type
+                    .as_accessor()
+                    .flag()
+                    .contains(FieldTypeFlag::UNSIGNED)
```

Comment:
There is a method `is_unsigned()`. e.g: `field_type.as_accessor().is_unsinged()`.

Reply:
I prefer to only make the necessary changes for supporting Prost. Trying to do more in this PR will make it harder to land and rebase

Reply:
I agree it would be an improvement though!

Reply:
OK